bin/rose-date

```diff
@@ -75,7 +80,10 @@
 #         Use the value of the ROSE_TASK_CYCLE_TIME environment variable as the
 #         reference time instead of the current time.
 #     --utc, -u
-#         Assume date time in UTC instead of local or other time zones.
+#         Assume date time in UTC instead of local or other time zones. 
+#     --as-total=TIME_FORMAT
+#         Used to express an ISO8601 duration in the specified time format,
+#         hours h, minutes m or seconds s.
```

Comment:
I think it is better if we document it upper case - that's what the code stores anyway, and it fits the standard better (PT1H, etc).